examples/preflight/sample-preflight.yaml

```diff
@@ -24,32 +18,101 @@ spec:
               when: ">= 1.15.0"
               message: Your cluster meets the recommended and required versions of Kubernetes.
     - customResourceDefinition:
-        customResourceDefinitionName: constrainttemplates.templates.gatekeeper.sh
-        checkName: Gatekeeper policy runtime
+        customResourceDefinitionName: ingressroutes.contour.heptio.com
```

Comment:
Would this only succeed with contour specifically? What if they have nginx or any number of other ingress controllers? For an example, I'm going checking only for contour, but maybe then the message should specifically say it requires Contour. Ultimately, I'd love a sample check that looks for all the controllers listed on https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/, but maybe the smallest change here is message to be Contour-specific. @marccampbell does that make sense?